meta-webos/recipes-graphics/libsdl2/libsdl2_%.bbappend

```diff
@@ -2,6 +2,6 @@
 
 EXTENDPRAUTO_append = "webos1"
 
-PACKAGECONFIG_class-native = "x11 opengl"
-PACKAGECONFIG_class-nativesdk = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'x11', '', d)}"
+PACKAGECONFIG_class-native = "wayland opengl"
+PACKAGECONFIG_class-nativesdk = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland', '', d)}"
```

Comment:
This change can affect qemu emulator.
So please check more carefully original native target dependency.
original dependencies: qemu-native -> libsdl2-native -> x11-native